database/README.md

```diff
@@ -309,6 +326,56 @@ Indexes:
 - PRIMARY KEY: btree(`id`)
 - quotes\_token\_expiration: btree (`sell_token`, `buy_token`, `expiration_timestamp` DESC)
 
+### proposed\_solutions
+
+All solutions reported by solvers, that were part of a solver competition.
+
+ Column        | Type      | Nullable | Details
+---------------|-----------|----------|--------
+ auction\_id   | bigint    | not null | auction for which the solution was proposed 
+ uid           | bigint    | not null | unique id of the proposed solution within a single auction
+ id            | bytea     | not null | id of the proposed solution as reported by the solver
+ solver        | bytea     | not null | solver submission address
+ is\_winner    | boolean   | not null | specifies if a solution was chosen as a winner of a competition
+ score         | numeric   | not null | score of a solution, based on a scoring criteria used at the time of competition
+ price\_tokens | bytea[]   | not null | tokens used in a solution, for which uniform prices are provided
+ price\_values | numeric[] | not null | uniform prices for all tokens in price\_tokens list
+
+Indexes:
+- PRIMARY KEY: btree(`auction_id`, `uid`)
+
+### proposed\_solution\_executions
+
+Contains all order executions for proposed solutions. 
+
+ Column         | Type     | Nullable | Details
+----------------|----------|----------|--------
+ auction\_id    | bigint   | not null | auction for which the order was executed 
+ solution_uid   | bigint   | not null | `uid` from proposed\_solutions
+ order_uid      | bigint   | not null | id of the order
+ executed\_sell | numeric  | not null | the effective amount that left the user's wallet including all fees
+ executed\_buy  | numeric  | not null | the effective amount the user received after all fees
+
+Indexes:
+- PRIMARY KEY: btree(`auction_id`, `solution_uid`, `order_uid`)
+
+### proposed\_jit\_orders
+
+Solvers report orders they solved on each competition. Orders that don't exist in the orderbook, are considered JIT orders and saved in this table.
```

Comment:
A bit more context on why a solver propose an unknown order would be nice. (e.g. provide private liquidity or settle a surplus capturing jit order trade)